src/components/Buttons/TextButton.tsx

```diff
@@ -3,15 +3,21 @@ import styled from 'styled-components';
 import { spacing } from '../variables';
 import { variants } from './variants';
 
-export type ButtonColorVariant = 'dark' | 'disabled' | 'light' | 'outline';
+export type ButtonVariant =
+  | 'danger'
+  | 'dark'
+  | 'disabled'
+  | 'light'
+  | 'outline'
+  | 'textOnly';
```

Comment:
instead of defining these, we could infer them from the variants-object with
```typescript
type ButtonVariant = keyof typeof variants
```
i like it better, because then the object is the source of truth

Reply:
Yes this is true! Less work for us when we don't have to update **both** that type and the object!

Reply:
Done! I also moved the type to the variants file.